backend/inspirehep/assign/views.py

```diff
@@ -64,9 +66,9 @@ def assign_to_author(from_author_recid, to_author_recid, literature_recids):
     unstub_author_by_recid(to_author_recid)
 
 
-@blueprint.route("", methods=["POST"])
+@blueprint.route("authors", methods=["POST"])
 @login_required_with_roles([Roles.cataloger.value])
-def assign_view():
+def author_assign_view():
```

Comment:
```suggestion
def authors_assign_view():
```

Reply:
But this is not bulk like action, we are assigning/replacing one author for another in multiple papers. I don't think that `authors` here will be correct.